api/nakadi-event-bus-api.yaml

```diff
@@ -1626,6 +1650,11 @@ definitions:
           Might assume the special name BEGIN, meaning a pointer to the offset of the oldest
           available event in the partition.
         type: string
+      unconsumed_events:
+        description: |
+          Number of events unconsumed by the client. This is also known as consumer lag and is used for monitoring
```

Comment:
I'd suggest documenting this as approximate if it's approximate, or mentioning that it might be out of step with what's exactly in the stream at the time the count is examined. The "latency physics" suggest by the time i read it, it's possibly gotten out of date for a high volume stream. SQS is an example here which goes as far as naming their metadata with "approximate" in some case, eg http://docs.aws.amazon.com/AWSSimpleQueueService/latest/APIReference/API_ReceiveMessage.html, but a sentence might be enough here.